server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -546,6 +547,31 @@ const accountMutations = {
       return collectivelib.deleteCollective(account);
     },
   },
+  sendMessage: {
+    type: new GraphQLObjectType({
+      name: 'SendMessageResult',
+      fields: {
+        success: { type: GraphQLBoolean },
+      },
+    }),
+    description: 'Send message to an account.',
+    args: {
+      account: {
+        type: new GraphQLNonNull(AccountReferenceInput),
+        description: 'Reference to the Account to send message to.',
+      },
+      message: {
+        type: new GraphQLNonNull(GraphQLString),
```

Comment:
:ice_cream: We started using this one recently, I think it's nice to have it

```suggestion
        type: new GraphQLNonNull(GraphQLNonEmptyString),
```